docs/tempo/website/grafana-agent/_index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Grafana Agent
+weight: 400
+---
+
+# Grafana Agent
+
+The [Grafana Agent](https://github.com/grafana/agent) is a telemetry 
+collector for sending metrics, logs, and trace data to the opinionated 
+Grafana observability stack.
+
+It is commonly used as a tracing pipeline, offloading traces from the 
+application and forward them to a storage backend.
+The Grafana Agent tracing stack is built using OpenTelemetry.
+
+The Grafana Agent supports receiving traces in multiple formats:
+OTLP (OpenTelemetry), Jaeger, Zipkin, OpenCensus and Kafka.
+It sends traces in OTLP format via gRPC.
+
+On top of receiving and exporting traces, the Grafana Agent contains many 
+features that make your distributed tracing system more robust, and 
+leverages all the data that is processed in the pipeline.
+
+## Architecture
```

Comment:
I thought that this section can be a bit confusing for a beginner. Mostly the thing about running a set of Tempo instances. The new diagram that we've on Getting Started is lovely... 

Maybe we can create one for here?

Reply:
Right, the naming is not ideal. We can change it to tracing instances or tracing pipelines. WDYT?

Reply:
I like pipelines :)

But no strong preference!

Reply:
Sry, let me nit you one last time:

> The Grafana Agent can run a set of tracing instances, each of which configures its own pipeline.

Maybe this:
The Grafana Agent can be configured to run a set of tracing pipelines to collect data from your applications and write it to Tempo.

Reply:
Fixed, thanks!